demisto_sdk/commands/validate/tests/PB_validators_test.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from demisto_sdk.commands.validate.tests.test_tools import create_playbook_object
+from demisto_sdk.commands.validate.validators.PB_validators.PB118_is_input_key_not_in_tasks import (
+    IsInputKeyNotInTasksValidator,
+)
+
+
+@pytest.mark.parametrize(
+    "input_in_task,input_in_section, expected_result",
+    [
+        ({"first_input":"inputs.input_name1"}, [{"key":"input_name1", "value":"input_value1"}], []),
+        ({"first_input":"inputs.input_name1", "second_input": "inputs.input_name2"}, [{"key":"input_name1", "value":"input_value1"}], []),
+         ({"first_input":"inputs.input_name1"}, [{"key":"input_name2", "value":"input_value2"}], "The playbook 'Detonate File - JoeSecurity V2' contains the following inputs that are not used in any of its tasks: input_name2")
+    ],
+)
+def test_is_valid_all_inputs_in_use(
+  input_in_task, input_in_section, expected_result
+):
+    """
+        Given:
+        - A playbook with inputs in some tasks and inputs defined in the inputs section
+            Scenario 1: All inputs defined in the inputs section are in use in the playbook
```

Comment:
Let's change Scenario to Case (for all files) as this is the terminology we usually use for our tests.

Reply:
Done.